### DISCUSSION TITLE
Mac中字体警告xxx does not contain requested(fontspec) Script "CJK"

### DISCUSSION THREAD

**Discussion by ZhouWenjun2019** [stone-zeng/fduthesis#273] [Q&A] [answered]
设置cjk-font=mac/fandol后，警告如下：

- cjk-font=mac

<img width="547" alt="mac" src="https://user-images.githubusercontent.com/48540421/197190623-f9cb3628-79d2-4213-b553-d9543ad994ab.png">

- cjk-font=fandol

<img width="547" alt="fandol" src="https://user-images.githubusercontent.com/48540421/197190643-8689bc8b-0012-44d4-9d5f-5598fc560166.png">
请问，这是mac固有的问题吗？怎么做才能改进？

**Comment by stone-zeng** [comment, marked as answer]
见 https://github.com/CTeX-org/forum/issues/34

**Reply by ZhouWenjun2019** [reply]
查看了issue，发现这个问题还是直接忽略比较好？会对文档有影响吗？

**Reply by xkwxdyy** [reply]
> 查看了issue，发现这个问题还是直接忽略比较好？会对文档有影响吗？

请你看完这个再提问 https://github.com/CTeX-org/forum/issues/34#issuecomment-1009289852

**Reply by stone-zeng** [reply]
> 查看了issue，发现这个问题还是直接忽略比较好？会对文档有影响吗？

没有影响，可以忽略。
